### DISCUSSION TITLE
Deleting sets and gestures

### DISCUSSION THREAD

**Discussion by Penthell** [OpenTails/CRUMPET-Android#143] [Q&A]
I've noticed the option for deleting copied sets and chosen wags in gestures doesnt work. The minus (it says delete when highlighted) in the sets' menu only closes the menu. For the gestures, the x arrow next to each chosen wag reacts when pressed, but doesnt remove the wag (or in this case, return it to "no command).

Im I selecting the options wrong or is this a bug thats is still being looked into?

**Comment by leinir** [comment]
Thanks for the report! Since i can't just turn this into an issue, i've created an issue, and closing this discussion - further stuff over here: https://github.com/OpenTails/CRUMPET-Android/issues/144 :)
